redbot/core/core_commands.py

```diff
@@ -457,11 +465,69 @@ def get_bot_uptime(self, *, brief: bool = False):
             await ctx.send("No exception has occurred yet")
 
     @commands.command()
-    @checks.is_owner()
-    async def invite(self, ctx: commands.Context):
+    @commands.check(CoreLogic._can_get_invite_url)
```

Comment:
This isn't a valid for use as a check. Will result in a `TypeError` at runtime. (d.py only passes context, and there's no source of `self` for the instance method)

You can write a check which uses `ctx.bot.get_cog("Core")._can_get_invite_url`, or refactor this to not be a check.